events/bingo/bingoBossApp/ui/css/bingoBossApp_ui.css

```diff
@@ -0,0 +1,109 @@
+*, *:before, *:after {
+    -webkit-box-sizing: inherit;
+    -moz-box-sizing: inherit;
+    box-sizing: inherit;
+}
+
+html {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Raleway', sans-serif;
+    color: #cccbcb;
+    background-color: #2b2b2b;
+    overflow: hidden;
+    margin: 0;
+    padding: 0;
+}
+
+#titleBarContainer {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    padding: 0 16px;
+    font-size: 24px;
+    background-color: #121212;
+    color: #ffffff;
+    justify-content: space-between;
+}
+
+#buttonsContainer {
+    margin: 0 8px;
+    height: calc(100vh - 60px - 180px);
+    display: flex;
+    justify-content: space-evenly;
+    flex-direction: column;
+}
+
+input[type=button] {
+    display: block;
+    font-family: 'Raleway';
```

Comment:
Do you need Raleway here if you defined it in the body above already?

Reply:
better to be explicit. if someone accidentally changed the body font, we don't want the button font to change.